.github/ISSUE_TEMPLATE/breaking-change.md

```diff
@@ -1,65 +1,80 @@
----
-name: ASP.NET Core breaking change
-about: Report a breaking change in ASP.NET Core
-
----
-<!--
-This issue template is for use in opening issues that document breaking changes. This template should be used to create an issue by Microsoft product team members who are documenting a breaking change.
-
-Bracketed text is a placeholder; replace the text with the requested information and remove the brackets before submitting the issue. Also, remove this comment before submitting the issue.
-
--->
-## [Change Title]
-
-[Brief description of the change]
-
-### Version introduced
-
-[Version in which the breaking change first occurred, including the preview number if applicable (for example, ASP.NET Core 6 Preview 6)]
-
-### Old behavior
-
-### New behavior
-
-### Reason for change
-
-### Recommended action
-
-[ Suggested steps if user is affected go here:
-- Possible workarounds
-- Example of code changes to handle change
-]
-
-### Category
-<!-- Choose one of the following compatibility categories: -->
-
-- [ ] Binary compatibility (the newer version can be used without recompiling)
-- [ ] Source compatibility (successfully recompiling against newer version requires source changes)
-
-### Affected APIs
-
-[ If no APIs are affected, this should read:
-     "Not detectable via API analysis"
-
-  If affected APIs are identifiable, include a link for each. The link takes the form:
-
-  `[friendly description of API](link to API on docs.microsoft.com)`
-
-  For example, `[String.IndexOf(String)](https://docs.microsoft.com/dotnet/api/system.string.indexof#System_String_IndexOf_System_String_)
-
-  For methods, if all overloads are affected, link to the general overloaded method page. For example:
-
-  <https://docs.microsoft.com/dotnet/api/system.string.indexof>
-
-  Otherwise, link to the individual method overload. For example:
-
-  <https://docs.microsoft.com/dotnet/api/system.string.indexof#System_String_IndexOf_System_String_>
-
-]
-
-<!-- Do not modify anything below this line -->
-
----
-#### Issue metadata
-
-* Issue type: breaking-change
+name: "ASP.NET Core breaking change"
+description: Report a change in ASP.NET Core that breaks something that worked in a previous version. Intended mostly for product-team use.
+title: "[Breaking change]: "
+labels: "Breaking change"
+assignees:
+ - gewarren
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Brief description of the breaking change.
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of .NET introduced the breaking change?
+      options:
+        - .NET 6 Preview 6
+        - .NET 6 Preview 7
+        - .NET 6 RC1
+        - .NET 6 RC2
+        - .NET 6 GA
```

Comment:
Should these only include RC1, RC2 and GA? I assume most of the preview7 breaking changes (if any) are already announced)